### DISCUSSION TITLE
NetBeans IDE source code main file

### DISCUSSION THREAD

**Discussion by gskshashi** [apache/netbeans#7079] [Q&A]
I am exploring the source code of NetBeans IDE. But I am not able to find the file that is executed when we open the IDE. It will be very helpful if anyone can provide information on how to understand the source code of NetBeans

**Comment by mbien** [comment]
main method would be here https://github.com/apache/netbeans/blob/master/platform/o.n.bootstrap/src/org/netbeans/Main.java
